Bluepill-cli/Bluepill-cli/Simulator/SimulatorMonitor.h

```diff
@@ -11,13 +11,22 @@
 #import "BPExecutionPhaseProtocol.h"
 #import "BPExitStatus.h"
 
+typedef NS_ENUM(NSInteger, SimulatorState) {
```

Comment:
This is for internal state tracking and isn't used externally... there's no reason to make it public.